content/docs/gto/get-started.md

```diff
@@ -0,0 +1,162 @@
+---
+description:
+  'Learn how you can use GTO to create Artifact Registry in Git repository'
+---
+
+# Get Started
+
+GTO helps you build a Artifact Registry out of your Git repository. It does so
+by creating Git tags of [special format](/doc/gto/user-guide) and managing
+[`artifacts.yaml` metafile](/doc/gto/user-guide). Since committing large files
+to Git is not a good practice, you shouldn't commit binary files to Git.
+Instead, use DVC, Git-lfs, or any method commit pointers to the binary files
+instead.
+
+This Get Started will walk you through basic GTO Artifact Registy concepts and
+actions you would like to do in the Artifact Registry.
+
+## Showing the current state
+
+Assuming GTO is already [installed](/doc/gto/install) in your active python
+environment, let's clone the example repo
+
+```cli
+$ git clone https://github.com/iterative/example-gto
+$ cd example-gto
+```
+
+This repo represents a simple example of Machine Learning Model Registry. Let's
+review it:
+
+```cli
+$ gto show
+╒══════════╤══════════╤════════╤═════════╤════════════╕
+│ name     │ latest   │ #dev   │ #prod   │ #staging   │
+╞══════════╪══════════╪════════╪═════════╪════════════╡
+│ churn    │ v3.1.1   │ v3.1.1 │ v3.0.0  │ v3.1.0     │
+│ segment  │ v0.4.1   │ v0.4.1 │ -       │ -          │
+│ cv-class │ v0.1.13  │ -      │ -       │ -          │
+╘══════════╧══════════╧════════╧═════════╧════════════╛
+```
+
+Here we have 3 models: `churn`, `segment` and `cv-class`. The latest versions of
+them are shown in the column named `latest`. The latest is selected as the one
+having the greatest [SemVer](https://semver.org).
+
+Model versions could be promoted to different stages. Here we have 3 of them:
+`dev`, `prod` and `staging`. When a model was never promoted to a stage, we see
+`-` in the field.
+
+## Registering versions and assigning stages
+
+GTO can [register version](/doc/gto/command-reference/register) of artifacts and
+[assign them to stages](/doc/gto/command-reference/assign). Both functionalities
+work in a similar way, so let's walkthough only one of them here.
+
+Let's suppose the version `v0.1.13` of `cv-class` was looking very promising,
+and now we want to promote it to `dev` to test it:
+
+```cli
+$ gto assign cv-class --version v0.1.13 --stage dev
+Created git tag 'cv-class#dev#1' that assigns stage to version 'v0.1.13'
+To push the changes upstream, run:
+    git push origin cv-class#dev#1
```

Comment:
Note: `cli` highlighter interprets `#` as comments (create issue in the theme repo?)

<img width="778" alt="image" src="https://user-images.githubusercontent.com/1477535/198147148-f7f9901a-b62c-4e96-b239-4e618da544dd.png">

Also, the comment is too long and requires horizontal scroll.

> Remember that you can sanity check docs via review apps 🙂

Reply:
Good idea! Created https://github.com/iterative/gatsby-theme-iterative/issues/104

Reply:
Thanks

Reply:
Do we know whether https://github.com/iterative/gatsby-theme-iterative/pull/115 has been ported to this repo's main branch?